### DISCUSSION TITLE
Why send a request to a proxy https://chatgpt-proxy.fly.dev?

### DISCUSSION THREAD

**Discussion by x281737830** [acheong08/ChatGPT#748] [Q&A] [answered]
Hi, Why the request is send to this URL? How can I make sure the respose is return from chat.openai.com?, Thanks
BASE_URL = environ.get("CHATGPT_BASE_URL") or choice(
    ["https://chatgpt-proxy.fly.dev/", "https://chatgpt-proxy2.fly.dev/"]
)

**Comment by acheong08** [comment, marked as answer]
> Why the request is send to this URL?

Bypass cloudflare

> How can I make sure the respose is return from chat.openai.com?, Thanks

Go to chat.openai.com and you'll see your conversations pop up

**Reply by x281737830** [reply]
Thanks. The bypass cloudflare is official, or is build by someone else?

**Reply by acheong08** [reply]
Built by me

**Reply by druphliu** [reply]
can Built by our self？
